src/common/feedGenerator.ts

```diff
@@ -502,7 +502,7 @@ export const tagFeedBuilder = (
   builder.andWhere((subBuilder) => whereTags([tag], subBuilder, alias));
 
 export const fixedIdsFeedBuilder = (
-  ctx: Context,
+  ctx: unknown,
```

Comment:
I needed to use it inside worker which does not have the `ctx` available. The `ctx` is not really used in that specific builder but since all builders have that kind of arguments footprint I did not wish to remove it and opted to make parameter `unknown`.

This allows me to just pass `{}` inside the worker call https://github.com/dailydotdev/daily-api/blob/6d963b9a7e6a95d38bef1f25c1514013c3f18542/src/workers/personalizedDigestEmail.ts#L130-L131 and if for ever reason `ctx` becomes used inside `fixedIdsFeedBuilder` typescript will detect worker usage and we will be able to supply or `Pick` specific part of `Context`.